tests/settings.py

```diff
@@ -9,23 +11,52 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = "-#t-d1kx#a_l#5dt6qxpn2w)i8qfbn=-kth6g3m@xl=wa!!1uh"
 
-DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": ":memory:",
-    }
-}
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+STATIC_DIR = os.path.join(BASE_DIR, 'openhub_django/static/')
 
 INSTALLED_APPS = [
     "django.contrib.auth",
     "django.contrib.contenttypes",
     "django.contrib.sites",
+    "django.contrib.staticfiles",
     "openhub_django",
+    "django_distill",
+]
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": os.path.join(BASE_DIR, 'db.sqlite3'),
+    }
+}
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+            os.path.join(BASE_DIR, 'openhub_django/templates'),
```

Comment:
Can't there a configuration something like if we install this project, we can use custom templates by configuring it in the settings. Or by default these templates will be used.